doc/clustering.md

```diff
@@ -90,10 +91,10 @@ storage or networks.
 
 Those typically cover:
 
- - Source device for a storage pool (leaving empty would create a loop)
- - Name for a ZFS zpool (defaults to the name of the LXD pool)
- - External interfaces for a bridged network (empty would add none)
- - Name of the parent network device for managed `physical` or `macvlan` networks (must be set)
+- Source device for a storage pool (leaving empty would create a loop)
+- Name for a ZFS zpool (defaults to the name of the LXD pool)
+- External interfaces for a bridged network (empty would add none)
+- Name of the parent network device for managed `physical` or `macvlan` networks (must be set)
```

Comment:
Is there a rule for `-` vs `*`? I see you changed `-` to `*` in other files.

Reply:
[MD006 - Consider starting bulleted lists at the beginning of the line](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md006---consider-starting-bulleted-lists-at-the-beginning-of-the-line)

Reply:
I was actually wondering about https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md004---unordered-list-style - so it's depending on each file.